proto/p4info/xform/examples/gen_p4info.sh

```diff
@@ -0,0 +1,11 @@
+if [ $# -ne 1 ]; then
+	echo "USAGE: $0 <p4 source file>"
+	exit 1
+fi
+
+echo "Generating $1.p4info.proto ..."
+p4c-bm2-ss --p4runtime-file $1.p4info.proto --p4runtime-format binary $1
```

Comment:
Could you mention in the Readme that `p4c-bm2-ss` needs to be installed and in the path? Unless you mentioned it already and I missed it :)

Reply:
Done.